semver/types.ts

```diff
@@ -76,3 +82,6 @@ export interface SemVerRange {
   // The outer array is OR while each inner array is AND
   ranges: SemVerRangeOr;
 }
+
+export type Range = [SemVerComparator, SemVerComparator];
+export type RangeSet = Range[];
```

Comment:
Is this change related to the main objective of this PR?

Reply:
sorry, this slipped in. reverted.